### PR TITLE
[Release v1.1.0-alpha.0] Bump Helm chart version to 1.1.0-alpha.0

### PR DESCRIPTION
**Pull request by maelvls**
As specified in the README's release process. I've also updated the README since it was vastly outdated; I've added how things will work once @inteon's changes ([Harbor](https://gitlab.com/venafi/vaas/delivery/harbor/-/merge_requests/48/diffs) and [Enterprise Builds](https://github.com/jetstack/enterprise-builds/pull/222)) will have gone through.

### CODE REVIEW

File: deploy/charts/venafi-kubernetes-agent/Chart.yaml [hunk 1]
```diff
@@ -3,5 +3,5 @@ name: venafi-kubernetes-agent
 description: |-
   The Venafi Kubernetes Agent connects your Kubernetes or Openshift cluster to the Venafi Control Plane.
 type: application
-version: 1.0.0
-appVersion: "v1.0.0"
+version: 1.1.0-alpha.0
```
Comment (inteon) [inline deploy/charts/venafi-kubernetes-agent/Chart.yaml:6]: These versions should just remain 0.0.0 in the repo. Makefile modules will overwrite them normally.

Comment (inteon) [reply deploy/charts/venafi-kubernetes-agent/Chart.yaml:6]: See https://github.com/jetstack/venafi-enhanced-issuer/blob/main/deploy/charts/venafi-enhanced-issuer/Chart.yaml

File: README.md [hunk 2]
```diff
@@ -92,72 +94,83 @@ The release process is semi-automated.
...
+>  └─ oci://quay.io/jetstack/charts/venafi-kubernetes-agent --version 1.1.0 (this project's GitHub Actions)
+>     └─ oci://us.gcr.io/jetstack-secure-enterprise/charts/venafi-kubernetes-agent (Enterprise Builds's GitHub Actions)
+>         ├──  oci://registry.venafi.cloud/charts/venafi-kubernetes-agent --version 1.1.0 (Harbor Replication)
+>         └─   oci://private-registry.venafi.cloud/charts/venafi-kubernetes-agent --version 1.1.0 (Harbor Replication)
+>     └─ oci://eu.gcr.io/jetstack-secure-enterprise/charts/venafi-kubernetes-agent (Enterprise Builds's GitHub Actions)
+>         └─  oci://private-registry.venafi.eu/charts/venafi-kubernetes-agent --version 1.1.0 (Harbor Replication)
+>  └─ quay.io/jetstack/venafi-agent:v1.1.0 (this project's GitHub Actions)
+>      ├──  registry.venafi.cloud/venafi-agent/venafi-agent:v1.1.0 (Harbor Replication)
+>      ├──  private-registry.venafi.cloud/venafi-agent/venafi-agent:v1.1.0 (Harbor Replication)
+>      └─  private-registry.venafi.eu/venafi-agent/venafi-agent:v1.1.0 (Harbor Replication)
```
Comment (maelvls) [inline README.md:155]: This is my understanding of how things will look like once you have made the changes to Harbor and Enterprise Builds' GitHub Actions cron job. Does this match your understanding? @inteon

File: deploy/charts/venafi-kubernetes-agent/Chart.yaml (hunk 1 above)
Comment (maelvls) [reply deploy/charts/venafi-kubernetes-agent/Chart.yaml]: Thanks. I've fixed this.

**Review by inteon — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (maelvls) [reply README.md]: We just went over this together, it's all good

**Pull request merged**